### PR TITLE
Add shuffle functionality

### PR DESCRIPTION
**Pull request by hayribakici**
This PR adds a toggle shuffle functionality. It resolves issue #16.

### CODE REVIEW

File: mopidy_pummeluff/actions/playback.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     'PlayPause',
     'Stop',
     'PreviousTrack',
-    'NextTrack',
+    'NextTrack'
```
Comment (domibarton) [inline mopidy_pummeluff/actions/playback.py]: @hayribakici can you leave the trailing comma there please? :) See https://development-guidelines.confirm.ch/guidelines/python.html:

> If you’re expanding both lists and you make a diff, you’ll notice that the changes of spam are much harder to tell than the changes of eggs. Therefor prefer the multi- over the single-line format. Also make sure you always add a trailing comma to the last element as well.

**Comment by domibarton** [comment]
Can you squash all the commits and follow [these guidelines](https://development-guidelines.confirm.ch/guidelines/git.html#commit-messages), especially the prefix and imperative mode.

E.g.:

```
FEATURE: Add shuffle functionality
```

**Comment by domibarton** [comment]
Forgot to say it: THANK YOU, awesome :)

File: mopidy_pummeluff/actions/playback.py (hunk 1 above)
Comment (hayribakici) [reply mopidy_pummeluff/actions/playback.py]: okay, 
it's just, that the linter was unhappy because of it.

**Pull request merged**